tests/test-war-smoke/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM ${jetty.test.image}
+ADD root /app
```

Comment:
I'm having an issue with this working. See: #85. I actually can't get the test to pass without adding `RUN chmod -R +r /app` to this Dockerfile.
Maybe we should stick with the WAR deployment without exploding it for now, because that's what we've always been supporting.